lib/MultiStats.js

```diff
@@ -85,7 +85,7 @@ class MultiStats {
 
 		const obj = this.toJson(options, true);
 
-		return Stats.jsonToString(obj, useColors);
+		return Stats.jsonToString(obj, useColors); //TODO: convert this to non-static method
```

Comment:
Let's ignore this for now and take care of this when finished the other problems.